fixed probs

adeniji19 opened the pull request.

adeniji19 committed:
fixed probs

adeniji19 committed:
fixed probs

adeniji19 committed:
hmm. attempt number 9154

adeniji19 committed:
hmm. attempt number 9154

churichard commented on tigerpath/static/app/css/app-style.css:
```diff
@@ -125,8 +125,19 @@ thead.semester-header {
+div.inline {
+  float:left;
+}
+.btn:hover, .btn:focus, .btn:active, .btn.active, .open>.dropdown-toggle.btn {
```
You should rename all of these `btn` back to `nav-btn`

churichard commented on tigerpath/templates/tigerpath/index.html:
```diff
@@ -40,17 +40,22 @@
-        <a class="btn nav-btn hidden" href="{% url 'logout' %}" id="logout-btn">Logout</a>
-        <a class="btn nav-btn" href="#" id="help-btn"><i class="far fa-question-circle"></i></a>
+        <div class="dropdown-container inline">
+        <a class="btn" href="#" id="settings-btn">Settings</a>
```
readd `nav-btn` to the classes for everything that has a btn

churichard commented on tigerpath/templates/tigerpath/index.html:
```diff
@@ -40,17 +40,22 @@
-        <a class="btn nav-btn" href="#" id="netid-btn" onclick="toggler('logout-btn');">{{ request.user.profile.nickname|default:request.user.username }}</a>
-        <a class="btn nav-btn hidden" href="{% url 'logout' %}" id="logout-btn">Logout</a>
-        <a class="btn nav-btn" href="#" id="help-btn"><i class="far fa-question-circle"></i></a>
+        <div class="dropdown-container inline">
```
indent everything properly in this div

churichard commented on tigerpath/templates/tigerpath/index.html:
```diff
@@ -122,8 +127,8 @@
     <!-- external scripts -->
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```
don't move this

adeniji19 committed:
correcting indents

adeniji19 committed:
correcting indents

adeniji19 committed:
Merge branch 'dev' into logout

danielleung27 committed:
Color changes (#64)

* Color changes

* typo

churichard committed:
Update webpage titles (#63)

churichard reviewed (changes requested):
don't merge yet, problems with settings

danielleung27 committed:
Fix hex numbers for color changes (#65)

bnehoran committed:
Fix color explanation in the tutorial (#67)

churichard committed:
Change user profile major to foreign key (#69)

danielleung27 committed:
Update scraper to not crash (#72)

Sometimes recal doesn't scrape a course that the 2nd scraper scrapes so the 2nd scraper throws an error when it tries to get the course.

danielleung27 committed:
Fix scraper bug (#73)

churichard committed:
Add features and gif to landing page, modify privacy policy (#74)

churichard committed:
Modify landing page, add beta signifier (#76)

churichard committed:
Update readme, change offered text, remove courses json (#75)

bnehoran committed:
Add collapsed course lists and additional info to verifier output (#71)

* Corrected ELE tests that were broken when the ELE requirements error was fixed.

* Added finding a subreq by path.

* Improved comment for find_requirement()

* Install jsonschema

* Include collapsed course lists and additional info in verifier req output.

* Add additional fields and force order in contacts field.

* Updated tests to account for additional fields

* Remove unnecessary find_requirement() function.

* Removed unneeded if clause and renamed _get_collapsed_course_and_dist_req_sets()

* Made comments more informative.

* add space after comma

churichard committed:
Fix course title padding in schedule (#78)

bnehoran committed:
Remove the course lists from the verifier output for speed. (#79)

adeniji19 committed:
dropdown without bootstrap

adeniji19 committed:
resolve merge conflicts
